pkgs/tools/archivers/p7zip/default.nix

```diff
@@ -22,40 +22,33 @@ stdenv.mkDerivation rec {
     '';
   };
 
-  # Default makefile is full of impurities on Darwin. The patch doesn't hurt Linux so I'm leaving it unconditional
-  postPatch = ''
```

Comment:
> prePatch was not being used due to no patches being present

There is only `postPatch` and the `patchPhase` runs the hooks unless overridden or disabled even when there are no patches:

https://github.com/NixOS/nixpkgs/blob/287805fcb77f914fb53add71eabfe274dd6e9f55/pkgs/stdenv/generic/setup.sh#L1200

Reply:
not sure what I did here :upside_down_face: reverted